certbot/plugins/util.py

```diff
@@ -6,6 +6,20 @@
 
 logger = logging.getLogger(__name__)
 
+def get_prefixes(path):
+    """Retrieves all possible path prefixes of a path, in descending order
+    of length. For instance,
+        /a/b/c/ => ['/a/b/c/', '/a/b/c', '/a/b', '/a', '/']
```

Comment:
nit: We should document the parameter and return value using Sphinx syntax so if we build the documentation for this file, it's nicely formatted. You can find an example of what this syntax looks like and a link to the general Sphinx documentation [here](https://certbot.eff.org/docs/contributing.html#coding-style). There are many other examples in the code base.

You can build the documentation locally by activating the virtual environment and running the command described [here](https://certbot.eff.org/docs/contributing.html#updating-the-documentation).